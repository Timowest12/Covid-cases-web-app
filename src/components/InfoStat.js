import React from 'react';

const InfoStat = (props) => (
  <div className="infoStatsBlock">
    <div className="infoStat">
      <div className="infoStatLeft">{props.metric}</div>
      <div className="infoStatright">{props.amount}</div>
    </div>
  </div>

);

export default InfoStat;
