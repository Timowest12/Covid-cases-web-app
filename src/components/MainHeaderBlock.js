import React from 'react'

const MainHeaderBlock = () => {
    return (
        <div className='mainHeaderBlock'>
            <div className="mainHeaderBlockLeft"><img src='/worldMap.svg' /></div>
            <div className="mainHeaderBlockRight">
                Covid stats by country
            </div>
        </div>
    )
}

export default MainHeaderBlock
