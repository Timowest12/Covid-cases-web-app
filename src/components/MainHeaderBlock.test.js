import React from 'react';
import renderer from 'react-test-renderer';
import InfoStat from './InfoStat';
import MainHeaderBlock from './MainHeaderBlock';

it('test if mainheaderblock renders correctly', () => {
  const tree = renderer.create(<MainHeaderBlock />).toJSON();
  expect(tree).toMatchSnapshot();
});
