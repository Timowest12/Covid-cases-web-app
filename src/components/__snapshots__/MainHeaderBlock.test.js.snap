// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test if mainheaderblock renders correctly 1`] = `
<div
  className="mainHeaderBlock"
>
  <div
    className="mainHeaderBlockLeft"
  >
    <img
      alt="worldmap"
      src="/worldMap.svg"
    />
  </div>
  <div
    className="mainHeaderBlockRight"
  >
    Covid stats for each country around the world
  </div>
</div>
`;
